lib/LaTeXML.pm

```diff
@@ -255,7 +256,6 @@ sub convert {
       my ($state) = @_;    # Remove current state frame
       $$opts{searchpaths} = $state->lookupValue('SEARCHPATHS'); # save the searchpaths for post-processing
       $state->popDaemonFrame;
-      $$state{status} = {};
```

Comment:
this was the only place I had to touch the actual daemon core logic.

In essence, instead of immediately reset-ing the status hash after a conversion run is completed, I now reset it before a fresh run is started. This way I can still reuse the hash for passing on the log to a parent converter object.